statsdaemon.go

```diff
@@ -101,6 +101,7 @@ var (
 	percentThreshold  = Percentiles{}
 	prefix            = flag.String("prefix", "", "Prefix for all stats")
 	postfix           = flag.String("postfix", "", "Postfix for all stats")
+	heartbeatFilePath = flag.String("heartbeat-file-path", "", "path to heartbeat file")
```

Comment:
What do you think of updating this help text to "heartbeat file to update after a successful write to graphite" to better describe when the heartbeat file is written to. We also should update the help output in `README.md`